lib/shopify-cli/forms/create_app.rb

```diff
@@ -0,0 +1,78 @@
+require 'shopify_cli'
+require 'uri'
+
+module ShopifyCli
+  module Forms
+    class CreateApp < Form
+      positional_arguments :name
```

Comment:
that's how you do this! I was trying to get something like this in Populate. Cool pattern.